ext/pcre/php_pcre.c

```diff
@@ -163,6 +167,9 @@ static void php_free_pcre_cache(zval *data) /* {{{ */
 {
 	pcre_cache_entry *pce = (pcre_cache_entry *) Z_PTR_P(data);
 	if (!pce) return;
+	if (pce->subpats_table) {
```

Comment:
I think it s better if free_subpats_table carries the burden to check it wdyt ?

Reply:
I personally don't like free functions that accept `NULL`, so I'd prefer it as-is.